src/gui/transferlistmodel.cpp

```diff
@@ -168,7 +168,7 @@ QVariant TransferListModel::headerData(int section, Qt::Orientation orientation,
         {
             switch (section)
             {
-            case TR_QUEUE_POSITION: return QChar('#');
+            case TR_QUEUE_POSITION: return u'#';
```

Comment:
This column is now shown as "35". Any char in place of # here returns a number as column name.

Reply:
Well, `QVariant` treats `char16_t` as numeric type so you still need to explicitly tell it that we want `QChar` or `QString`:
`case TR_QUEUE_POSITION: return QChar(u'#');` or `case TR_QUEUE_POSITION: return u'#'_qs;`

Reply:
thanks, will fix.